recipe_scrapers/_utils.py

```diff
@@ -65,21 +73,35 @@
 
 def _extract_fractional(input_string: str) -> float:
     input_string = input_string.strip()
-    if "/" in input_string:
-        # for example "1 1/2" is matched
-        components = input_string.split(" ")
-        whole_part, fractional_part = components[0], components[-1]
+
+    # Handling mixed numbers with unicode fractions e.g., '1⅔'
+    for unicode_fraction, value in FRACTIONS.items():
+        if unicode_fraction in input_string:
+            parts = input_string.partition(unicode_fraction)
+            whole_number_part = parts[0]
+            fraction_part = value
```

Comment:
Possibly some small cleanups achievable here:

```suggestion
    for unicode_fraction, fractional_part in FRACTIONS.items():
        if unicode_fraction in input_string:
            whole_number_part, _, _ = input_string.partition(unicode_fraction)
```